Include/internal/pycore_ast.h

```diff
@@ -429,6 +430,10 @@ struct _expr {
             asdl_comprehension_seq *generators;
         } GeneratorExp;
 
+        struct {
+            expr_ty source;
```

Comment:
Maybe name `source` as `target` to be uniform with the struct for Delete?

Reply:
I prefer the name `source`, to distinguish it from the del. I also feel like `target` can be ambiguous in this case. If I look at this code:

```py
x = move y
```

I'd call `y` the *source* and critically `x` the *target* of the move. So for people not knowing all the implementation details, it might be confusion to read `e->v.Move.target` and have it refer to the *source* of the value.